digdag-core/src/main/java/io/digdag/core/Limits.java

```diff
@@ -1,13 +1,24 @@
 package io.digdag.core;
 
+import io.digdag.client.config.Config;
+
+import javax.inject.Inject;
+
 public class Limits
```

Comment:
I wonder that class name could be WorkflowExecutorConfig because agents won't be able to use this, precisely speaking (commented below).